packages/rocketchat-lib/server/lib/sendNotificationsOnMessage.js

```diff
@@ -72,6 +72,31 @@ function notifyUser(userId, user, message, room, duration) {
 	});
 }
 
+function notifyAudioUser(userId, user, message, room) {
+
+	const UI_Use_Real_Name = RocketChat.settings.get('UI_Use_Real_Name') === true;
+	message.msg = parseMessageText(message, userId);
+
+	if (UI_Use_Real_Name) {
+		message.msg = replaceMentionedUsernamesWithFullNames(message.msg, message.mentions);
+	}
+	let title = UI_Use_Real_Name ? user.name : `@${ user.username }`;
+	if (room.t !== 'd' && room.name) {
+		title += ` @ #${ room.name }`;
+	}
+	RocketChat.Notifications.notifyUser(userId, 'audioNotification', {
+		title,
+		text: message.msg,
```

Comment:
well i don't know about this part. I just changed the notification -> 'audioNotification'